<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RtSa_Dialog_SweepSetup</class>
 <widget class="QWidget" name="RtSa_Dialog_SweepSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*******************indoor**************/
#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
{
	border:1px;
	border-top-style:solid;
	border-top-color:black;
}
#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
#widget
{
	background-color:rgba(211,211,211,0.9);
}
#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
#widget&gt;QPushButton
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}
#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
#widget&gt;QPushButton::checked
{
    color:rgba(87, 87, 87,1);
}
#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
#widget&gt;QPushButton::!checked
{
}
#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
RtSa_LineEdit
{
	background-color:rgba(255,255,255,0.6);
	color:rgba(162, 115, 0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
#widget&gt;QLabel
{
font: 75 13px &quot;Arial&quot;;
}
#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
RtSa_LineEdit:focus
{
  outline: none;
  border:2px solid rgba(162,115,0,1);
}
#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
#widget&gt;#pb_Sweep_Once
{
	background-color:rgb(0,0,0);
	color:rgb(255,255,255);
}
/***************outdoor*****************/
#RtSa_Dialog_SweepSetup[status = &quot;outdoor&quot;]
{
	border:1px;
	border-top-style:solid;
	border-top-color:black;
}
#RtSa_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#widget
{
	background-color:rgba(211,211,211,0.9);
}
#RtSa_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#widget&gt;QPushButton
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}
#RtSa_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#widget&gt;QPushButton::checked
{
    color:rgba(87, 87, 87,1);
}
#RtSa_Dialog_SweepSetup[status = &quot;outdoor&quot;]
RtSa_LineEdit
{
	color:rgba(162, 115, 0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold; 
}
#RtSa_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;; 
   color:rgba(46,46,46,1);
}
#RtSa_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
#RtSa_Dialog_SweepSetup[status  =  &quot;outdoor&quot;] RtSa_LineEdit:disable
{ 
    color:rgba(133,133,133,0.6); 
}
#RtSa_Dialog_SweepSetup[status  =  &quot;outdoor&quot;] RtSa_LineEdit:focus
 { 
    outline: none;
	border:2px solid rgba(162,115,0,1); 
}
#RtSa_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#widget&gt;#pb_Sweep_Once
{
	background-color:rgb(0,0,0);
	color:rgb(255,255,255);
}
/***************night*****************/
#RtSa_Dialog_SweepSetup[status = &quot;night&quot;]
{
	border:1px;
	border-top-style:solid;
	border-top-color:black;
}
#RtSa_Dialog_SweepSetup[status = &quot;night&quot;]
#widget
{
	background-color:rgba(208,182,139,0.9);
}
#RtSa_Dialog_SweepSetup[status = &quot;night&quot;]
#widget&gt;QPushButton
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}
#RtSa_Dialog_SweepSetup[status = &quot;night&quot;]
#widget&gt;QPushButton::checked
{
   color:rgba(87, 87, 87,1);
}
/*#RtSa_Dialog_SweepSetup[status = &quot;night&quot;]
#widget&gt;QPushButton::!checked
{
   color:rgba(0, 0, 0, 1)
}
*/
#RtSa_Dialog_SweepSetup[status = &quot;night&quot;]
RtSa_LineEdit
{
	background: rgba(251,220,167,0.60);
   color:rgba(159,98,0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#RtSa_Dialog_SweepSetup[status  =  &quot;night&quot;] 
RtSa_LineEdit:disable
{
	color:rgba(133,133,133,0.6);
}

#RtSa_Dialog_SweepSetup[status  =  &quot;night&quot;] 
RtSa_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
}
#RtSa_Dialog_SweepSetup[status = &quot;night&quot;]
#widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
}
#RtSa_Dialog_SweepSetup[status = &quot;night&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
#RtSa_Dialog_SweepSetup[status  =  &quot;night&quot;] RtSa_LineEdit:disable
{ 
    color:rgba(133,133,133,0.6);
}
#RtSa_Dialog_SweepSetup[status  =  &quot;night&quot;] RtSa_LineEdit:focus
 { 
	outline: none;
	border:2px solid rgba(162,115,0,1);
}
#RtSa_Dialog_SweepSetup[status = &quot;night&quot;]
#widget&gt;#pb_Sweep_Once
{
	background-color:rgb(0,0,0);
	color:rgb(255,255,255);
}</string>
  </property>
  <property name="status" stdset="0">
   <string>night</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>252</width>
     <height>648</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>85</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sweep Type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_single">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>125</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_single:checked
{
	padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
	background-position:right;
}

#pb_single:!checked
{
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
	background-position:right;
}</string>
    </property>
    <property name="text">
     <string>Single</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_conti">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>125</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*#QPushButton::pressed
{
background-color: rgb(255, 188, 0);
}
*/
#pb_conti:checked
{
	padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
}
#pb_conti:!checked
{
padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
}
</string>
    </property>
    <property name="text">
     <string>Conti.</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>46</y>
      <width>212</width>
      <height>2</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cancel">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
#pb_cancel
{
background-color:rgba(0,0,0,0);
background-image: url(:/rtsa/image/btn_close_normal.png);
}
#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
#pb_cancel:pressed
{
background-image: url(:/rtsa/image/btn_close_click.png);
}

#RtSa_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#pb_cancel
{
background-color:rgba(0,0,0,0);
background-image: url(:/rtsa/image/btn_close_normal.png);
}
#RtSa_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#pb_cancel:pressed
{
background-image: url(:/rtsa/image/btn_close_click.png);
}

#RtSa_Dialog_SweepSetup[status = &quot;night&quot;]
#pb_cancel
{
background-color:rgba(0,0,0,0);
border-image: url(:/rtsa/image/night/btn_close_normal.png);
}
#RtSa_Dialog_SweepSetup[status = &quot;night&quot;]
#pb_cancel:pressed
{
border-image: url(:/rtsa/image/night/btn_close_click.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>19</y>
      <width>102</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}</string>
    </property>
    <property name="text">
     <string>Sweep</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Sweep_Once">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>180</y>
      <width>216</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
#pb_Sweep_Once
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color: rgba(255, 255, 255, 1);	
	background-image: url(:/rtsa/image/btn_w216_nor.png);
}
#RtSa_Dialog_SweepSetup[status = &quot;indoor&quot;]
#pb_Sweep_Once:pressed
{
	background-image: url(:/rtsa/image/btn_w216_click.png);
}

#RtSa_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#pb_Sweep_Once
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color: rgba(0, 0, 0, 1);	
	background-image: url(:/rtsa/image/outdoor/btn_w216_nor.png);
}
#RtSa_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#pb_Sweep_Once:pressed
{
	background-image: url(:/rtsa/image/btn_w216_click.png);
}

#RtSa_Dialog_SweepSetup[status = &quot;night&quot;]
#pb_Sweep_Once
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color: #FBDBA7;
	background-image: url(:/rtsa/image/btn_w216_nor.png);
}
#RtSa_Dialog_SweepSetup[status = &quot;night&quot;]
#pb_Sweep_Once:pressed
{
	background-image: url(:/rtsa/image/btn_w216_click.png);
}</string>
    </property>
    <property name="text">
     <string>Sweep Once</string>
    </property>
   </widget>
   <widget class="RtSa_LineEdit" name="leAcqtime">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>121</width>
      <height>35</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAcqMan">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>290</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pbAcqMan:checked
{
	padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
	background-position:right;
}

#pbAcqMan:!checked
{
   padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
	background-position:right;
}</string>
    </property>
    <property name="text">
     <string>Man</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">AcqTimeGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pbAcqAuto">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>290</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pbAcqAuto:checked
{
	padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
}
#pbAcqAuto:!checked
{
padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
}
</string>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">AcqTimeGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>350</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Acq Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>290</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AcqTime </string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_Auto_Density">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>520</y>
     <width>216</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pb_Auto_Density
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color: rgba(255, 255, 255, 1);	
	background-image: url(:/rtsa/image/btn_w216_nor.png);
}

#pb_Auto_Density:pressed
{
	background-image: url(:/rtsa/image/btn_w216_click.png);
}</string>
   </property>
   <property name="text">
    <string>Auto Adjust Density</string>
   </property>
  </widget>
  <widget class="RtSa_LineEdit" name="edit_curve">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>470</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Persistence</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Infinite">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>240</y>
     <width>69</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pb_Infinite:checked
{
	padding: 8px 14px 7px 15px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
}
#pb_Infinite:!checked
{
padding: 8px 14px 7px 11px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
}</string>
   </property>
   <property name="text">
    <string>Infinite</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="RtSa_LineEdit" name="edit_time">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="RtSa_LineEdit" name="edit_lowest">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>420</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>450</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Curve</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>335</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Highest Density</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>196</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Denisity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Finite">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>240</y>
     <width>69</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pb_Finite:checked
{
	padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_right.png);
	background-position:right;
}

#pb_Finite:!checked
{
padding: 8px 5px 7px 6px;
	background-image: url(:/rtsa/image/fre_checked_left.png);
	background-position:right;
}</string>
   </property>
   <property name="text">
    <string>Finite</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="RtSa_LineEdit" name="edit_highest">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>360</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>280</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Persistence Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>395</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Lowest Density</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RtSa_LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">rtsa_lineedit.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="AcqTimeGroup"/>
 </buttongroups>
</ui>
